packages/PackageBuilder/tests/Strings/StringFormatConverterTest.php

```diff
@@ -30,6 +30,8 @@ public function provideCasesForCamelCaseToUnderscore(): Iterator
     {
         yield ['hiTom', 'hi_tom'];
         yield ['GPWebPay', 'gp_web_pay'];
+        yield ['GPWebPay', 'gp_web_pay'];
```

Comment:
Why is this twice here?

Reply:
My bad. 🤦‍♂ Fixed in 847e72c38a8c34a5cc0232c3caec5395ac7e1ad7.